.github/workflows/manual-build.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
     secrets: inherit
 
   build-docker:
-    needs: [ set-up-job, build-code-ubuntu2404 ]
+    needs: [ set-up-job, build-code-ubuntu2404, test-code ]
```

Comment:
do we want this this? Sometimes we want some builds on feature branches that work is not complete and tests might fail

Reply:
isn't this does this opposite? makes build-docker need to test the code?